srcpkgs/python3-pathspec/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-pathspec'
 pkgname=python3-pathspec
-version=0.8.0
+version=0.9.0
 revision=1
 wrksrc="pathspec-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3"
 short_desc="Python3 utility library for pattern matching of file paths"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MPL-2.0"
-homepage="https://github.com/cpburnz/python-path-specification"
+homepage="https://pypi.org/project/pathspec"
```

Comment:
I meant, *different* domains increase chance of finding upstream if it moves. Removing everything but pypi doesn't help there.

Reply:
Thanks!